### PR TITLE
MSK event source mapping examples

### PR DESCRIPTION
**Pull request by nikhilpenmetsa**
This project provides examples of filtering options available for Amazon MSK event source mapping. This work is part of a Serverless TFC's research initiative to help customers with building rules.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by nikhilpenmetsa** [comment]
Hello @mavi888 . Is there any action pending on my side?

File: lambda-esm-msk-filters-sam/README.md [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+Login to the client machine in the bastion environment that is configured to publish messages to the topic and run these commands.
+
+```
+#setting broker info
+export BOOTSTRAP_SERVERS_CONFIG="enter cluster's bootstrap brokers"
+#Instructions - https://docs.aws.amazon.com/msk/latest/developerguide/msk-get-bootstrap-brokers.html
+#Sample server config
+#export BOOTSTRAP_SERVERS_CONFIG="b-1.mskclustermsk.abcdef.c14.kafka.us-east-1.amazonaws.com:9092,b-3.mskclustermsk.abcdef.c14.kafka.us-east-1.amazonaws.com:9092,b-2.mskclustermsk.abcdef.c14.kafka.us-east-1.amazonaws.com:9092"
+
+#set topic name
```
Comment (mavi888) [inline lambda-esm-msk-filters-sam/README.md:303]: you are missing the step of creating the topic 

$ bin/kafka-topics.sh --create --topic TopicName --bootstrap-server $BOOTSTRAP_SERVERS_CONFIG

**Comment by mavi888** [comment]
This pattern is very well documented. Just fix that small thing there and i can merge it. 

Thanks

File: lambda-esm-msk-filters-sam/README.md (hunk 1 above)
Comment (nikhilpenmetsa) [reply lambda-esm-msk-filters-sam/README.md:306]: I have added this step now. Thanks for reviewing.

**Comment by nikhilpenmetsa** [comment]
Hi @mavi888 . I have incorporated the instructions for creating the topic. Can you please review, and if appropriate merge?

**Comment by mavi888** [comment]
Thanks for submitting your pattern.
Now this pattern is public in Serverless land

https://serverlessland.com/patterns/lambda-esm-msk-filters-sam

You can share your pattern using this link :)

**Pull request merged**